Dynatrace/dynatrace-operator
# Fix PodDisruptionBudget not obeying highAvailability flag

## Description

Ticket: https://dt-rnd.atlassian.net/browse/DAQ-2140

Fixes https://github.com/Dynatrace/dynatrace-operator/issues/4008

Currently the highAvailability flag does not control the creation of the PodDisruptionBudget

## How can this be tested?

set highAvailability to false in values file -> make manifest -> check if the PDB is there
